### ISSUE TITLE
About DCGAN Loss

### ISSUE THREAD

**Issue by MTandHJ** [POSTECH-CVLab/PyTorch-StudioGAN#67]
Hi,

It's a nice and helpful work, implementing a collections of useful tricks of GAN.
I feel confused about the implementation of loss_dcgan in src/utils./losses.py.
Is the loss_dcgan just a reproduction of BCELoss?
If so, I think 
`dis_loss = -torch.mean(nn.LogSigmoid()(dis_out_real) + (ones - dis_out_fake.sigmoid()).log()`
may be more reasonable, or a safer one
`dis_loss = -torch.mean(nn.LogSigmoid()(dis_out_real) + (ones - dis_out_fake.sigmoid() + 1e-6).log()`

Best,
MTandHJ

**Comment by MINGUKKANG** [comment]
Hi MTandHJ,

Thank you very much:)

I have reflected your modification above. 

Best,

Minguk

**Comment by MINGUKKANG** [comment]
Thank you.

I reflect your suggestion.

Sorry for the late response.
